cell-toolbar/pyproject.toml

```diff
@@ -5,8 +5,8 @@ build-backend = "hatchling.build"
 [project]
 name = "jupyterlab_examples_cell_toolbar"
 readme = "README.md"
-license = {text = "BSD-3-Clause License"}
-requires-python = ">=3.8"
+license = { file = "LICENSE" }
```

Comment:
CI is now failing with:

```
  OSError: License file does not exist: LICENSE
```

Probably was generated by template but not added with `git add`

Reply:
Thank you @krassowski !
I ran again the copier update: the `LICENSE` files quickly appeared in the explorer of my codium then disappeared.
I don't know what happened exactly, as when creating an extension for the first time with the latest template the `LICENSE` file is there.
Anyway I have manually added all the `LICENSE` files.